.zenodo.json

```diff
@@ -0,0 +1,24 @@
+{
+    "creators": [
```

Comment:
Should all authors in `DESCRIPTION` be listed here or? There is also the contributor field if we want to use that?